### PR TITLE
fix(perf): reduced initial start time

### PR DESCRIPTION
**Pull request by alexander-akait**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

No need

### Motivation / Use-Case

Start faster

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -355,7 +381,7 @@ class Server {
    */
   static internalIPSync(family) {
     try {
-      const { gateway } = defaultGateway[family].sync();
+      const { gateway } = require("default-gateway")[family].sync();
```
Comment (alexander-akait) [inline lib/Server.js:384]: We use only one method or async or sync, so it is fine to use `require`

Comment (alexander-akait) [reply lib/Server.js:384]: Also we don't it when a developer provide IP, so avoid it for some cases

File: lib/Server.js [hunk 2]
```diff
@@ -210,6 +208,34 @@ if (!process.env.WEBPACK_SERVE) {
...
+    // Allow to clean up memory for fn
+    // and all dependent resources
+    // eslint-disable-next-line no-undefined
+    fn = undefined;
+
+    return result;
+  };
+};
+
+const getExpress = memoize(() => require("express"));
```
Comment (alexander-akait) [inline lib/Server.js:237]: Yeah, we always run `express`, but let's do it lazy

**Review by snitin315 — APPROVED** [review]
Looks good.

**Pull request merged**